### PR TITLE
Prep for v1.25.2

### PR DESCRIPTION
**Pull request by odow**
## TODO

- [x] https://github.com/jump-dev/MathOptInterface.jl/pull/2409

## Basic

 - [x] `version` field of `Project.toml` has been updated
       - If a breaking change, increment the MAJOR field and reset others to 0
       - If adding new features, increment the MINOR field and reset PATCH to 0
       - If adding bug fixes or documentation changes, increment the PATCH field

## Documentation

 - [x] Add a new entry to `docs/src/changelog.md`, following existing style

## Tests

 - [x] https://github.com/jump-dev/MathOptInterface.jl/actions/runs/7633463206

### CODE REVIEW

**Comment by odow** [comment]
There is one new failure in Cbc that is expected, but worth exploring
image

File: docs/src/changelog.md [hunk 1]
```diff
@@ -7,6 +7,23 @@ CurrentModule = MathOptInterface
...
+   (#2396)
+ - Fixed `modify_function!` for [`ScalarQuadraticCoefficientChange`](@ref)
+   (#2408)
+ - Fixed writing `FileFormats.MOF` files if [`ScalarNonlinearFunction`](@ref)
+   contains [`ScalarAffineFunction`](@ref) or [`ScalarQuadraticFunction`](@ref)
+   (#2409)
+
+### Other
+
+ - Clarified ordering of nonlinear tape in documentationn (#2401)
```
Comment (odow) [inline docs/src/changelog.md:24]:
```suggestion
 - Clarified ordering of nonlinear tape in documentation (#2401)
```

**Comment by blegat** [comment]
> There is one new failure in Cbc that is expected, but worth exploring

You can remove the `ZerosBridge` as already done by a few other solvers, e.g.,
https://github.com/jump-dev/CSDP.jl/blob/6f7ea93d9a443f4c4396eaca117aefd40e1a13fc/test/MOI_wrapper.jl#L54

**Comment by odow** [comment]
Yeah, we could work around this in the Cbc tests, but the issue is that we aren't throwing the correct error so things don't work as expected. See #2413

**Comment by odow** [comment]
I removed ZerosBridge from Cbc so it's now passing tests. I'll merge and tag this once CI finishes. #2413 can wait for a longer discussion.

**Pull request merged**